app/src/main/res/layout/library_site_item.xml

```diff
@@ -77,9 +78,10 @@
             android:layout_height="@dimen/glyph_button_height"
             android:background="?android:attr/selectableItemBackgroundBorderless"
             android:contentDescription="@string/content_description_menu"
-            app:srcCompat="@drawable/ic_menu"
+            app:srcCompat="@drawable/ic_close"
```

Comment:
We might want to update the id here since it's a close button now instead of an overflow menu.

I will also note that layout width/height is also unusual being 40x40 instead of 24x24. Normally, I would think about just changing this to 24x24, but then we might need to call `increaseTapArea` so maybe that was the consideration done here. In Compose, we were able to leave the icon buttons as 24x24 because the underlying internal compose implementation will ensure that the tap area satisfies accessibility requirements.